packages/framework/presence/src/presence.ts

```diff
@@ -93,6 +93,11 @@ export interface ISessionClient<
 	 *
 	 */
 	getConnectionStatus(): SessionClientStatus;
+
+	/**
+	 * Set client connection status to disconnected.
+	 */
+	setDisconnected(): void;
```

Comment:
What is the scenario where customer should be updating a session client's status? (I expect that there isn't.)